test/acceptance/profile_test.rb

```diff
@@ -29,8 +29,10 @@ def test_profile_shows_progress
 
   def test_display_team_invites_only_in_users_own_profile
     new_user = create_user(username: 'new_user', github_id: 456)
-    attributes = { slug: 'some-team', name: 'Some Name', usernames: 'new_user' }
-    Team.by(@user).defined_with(attributes, @user).save!
```

Comment:
Hi, yes (sorry! it's taken forever to catch up with everything).

Looking now.